README.md

```diff
@@ -1,67 +1,128 @@
-# Ruby Course
+# 📗 Table of Contents
 
-If you are not familiar with linters and GitHub Actions, read [root level README](../README.md).
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+## 📖 OOP-School-Library<a name="about-project"></a>
 
-## Set-up Rubocop GitHub Action
+In this project, I achieved the following
 
-[Rubocop](https://www.rubocop.org/) is a Ruby static code analyzer (a.k.a. linter) and code formatter. It will enforce many of the guidelines outlined in the community [Ruby Style Guide](https://rubystyle.guide/).
+- Created Class Person
+- Created Class Student
+- Created Class Teacher
```

Comment:
- [ ] In the teacher class, to avoid errors from the `initialize` method you need to use the correct notation for assigning `parent_permission`. Kindly change the **#super** method from this:

  ```
  super(age, name, parent_permission)
  ```
  to this:
  ```
  super(age, name, parent_permission: parent_permission)
  ```